examples/gallery/plot/image.py

```diff
@@ -4,9 +4,9 @@
 The :meth:`pygmt.Figure.image` method can be used to read and
 place a raster image file or an Encapsulated PostScript file
 on a map. We must specify the file as *str* via the ``imagefile``
-argument or simply use the filename as the first argument. You can
+parameter or simply use the filename as the first parameter. You can
```

Comment:
```suggestion
parameter, or simply use the filename as the first argument. You can
```
I feel like this sentence is repeating itself somehow. The str input is an argument, which is passed to the `imagefile` parameter. Alternatively, the filename (also a str) can be passed in as an argument to the function?!

Reply:
You're right, thanks!